main.cpp

```diff
@@ -0,0 +1,66 @@
+#include "orbit.cpp"
+#include <stdio.h>
+#include<iostream>
+
+int main() {
+    ShipPosition initialPosition;
+    initialPosition.position.x = 0.0;
+    initialPosition.position.y = 6378.2;
+    initialPosition.position.z = 0.0;
+    initialPosition.orientation.x = 1.0;
+    initialPosition.orientation.y = 0.0;
+    initialPosition.orientation.z = 0.0;
+    initialPosition.speedFirst.x = 0.0;
+    initialPosition.speedFirst.y = 0.0;
+    initialPosition.speedFirst.z = 0.0;
+    initialPosition.moment.rotationAroundX = 0.0;
+    initialPosition.moment.rotationAroundY = 0.0;
+    initialPosition.moment.rotationAroundZ = 0.0;
+    ShipParams shipParams;
```

Comment:
Separate initializations of different structures by empty lines - for visibility:)